rust/projects/tools/project.md

```diff
@@ -114,19 +234,199 @@ this case the tests in `tests.rs` (`--test tests`). We can even narrow our testi
 down to a single test case within the `tests` test suite, like
 
 ```
-cargo test --test tests -- TODO
+cargo test --test tests -- cli_no_args
 ```
 
-And that's probably how you will be running the tests yourself as you work
+or, by matching multiple test cases, a few, like:
+
+
+```
+cargo test --test tests -- cli
+```
+
+_Try it now._
+
+That's probably how you will be running the tests yourself as you work
 through the project, otherwise you will be distracted by the many failing tests
-that you are not currently working on fixing.
+that you have not yet fixed.
 
 **Question D**: even if a given test suite doesn't contain any tests, why
 might we not want them to run? Besides issuing the above command, how could
```

Comment:
Do you mean "why might we want to to run?" here

Reply:
No, I mean "not", but I can see how the sentence suggests the opposite. I'll change it to just "Why might we not want to run empty test suites". The only answer I'm thinking of has to do with compile times. This question may be too obscure to keep.

Reply:
I still feel the question a bit confusing. Can we add something to lead readers to thinking about the compile time? For example, add "It takes no time to run an empty test.` to the beginning?